workflow/rules/variant_annotator.smk

```diff
@@ -4,33 +4,33 @@ __email__ = "massimiliano.volpe@liu.se"
 __license__ = "GPL-3"
 
 
-rule variant_annotator:
+rule annotation_variant_annotator:
```

Comment:
why add prefix "annotation_"

Reply:
Because I forgot to match Lauri's initial idea of naming the rule by starting from the name of the directory that stores the actual .smk file in Joel's repository. He started naming the rules like this so that you can easily recognise where in [pgx_module/src/](https://github.com/JoelAAs/pgx_module/tree/master/src) the original rules are located. Is it better to use a different system, for example using the exact same name that Joel assigned to the rule?

Reply:
ok! Will it not be a bit confusing since you now are storing the files directly under rules? The hydra-genetics best practice would be to have as you had, before the change. Which is to store everything directly under rules, gather rules that use the same tool in the same smk file (or if they are only used together)

Reply:
Oh yes, that's right. I think I can revert it back then, and I can modify the old rules as well. For example, annotation_variant_annotator will become variant_annotator. It's just a few rules right now so it should be manageable.